migration/testdata/v1_Trackstore.post

```diff
@@ -1,5 +1,5 @@
-/ffs/scheduler/tstore/QmY7gN6AfKSoR7DNEjcUyXRYS85giD1YXN62cWVzS5zfus,[{"IID":"ad2f3b0c-e356-43d4-a483-fba79479d7e4","StorageConfig":{"Hot":{"Enabled":false,"AllowUnfreeze":true,"UnfreezeMaxPrice":50000000,"Ipfs":{"AddTimeout":300}},"Cold":{"Enabled":true,"Filecoin":{"RepFactor":8,"DealMinDuration":518400,"ExcludedMiners":null,"TrustedMiners":null,"CountryCodes":null,"Renew":{"Enabled":false,"Threshold":0},"Addr":"t3taeln7s4dwgr42kvrajoqxuu3kfmh4vdf23xadnkjsxf2m5bnkof7shktll3es4sviytidzcmo572ibg4uvq","MaxPrice":500000000,"FastRetrieval":true,"DealStartOffset":8640}},"Repairable":true}},{"IID":"fb79f525-a3c4-47f0-94e5-e344c5b1dec5","StorageConfig":{"Hot":{"Enabled":true,"AllowUnfreeze":true,"UnfreezeMaxPrice":50000000,"Ipfs":{"AddTimeout":400}},"Cold":{"Enabled":true,"Filecoin":{"RepFactor":9,"DealMinDuration":518400,"ExcludedMiners":null,"TrustedMiners":null,"CountryCodes":null,"Renew":{"Enabled":false,"Threshold":0},"Addr":"t3taeln7s4dwgr42kvrajoqxuu3kfmh4vdf23xadnkjsxf2m5bnkof7shktll3es4sviytidzcmo572ibg4uvq","MaxPrice":500000000,"FastRetrieval":true,"DealStartOffset":8640}},"Repairable":true}}]
```

Comment:
The problem isn't much the system, but much of the migration logic per se.

To let it be fully backward compatible, on every migration we should always use copied structures of the original version. If we use structs directly from the current version, that's faster but not future-compatible.. since as we keep changing powergate, old migrations still reference the "current" structs (not really totally-frozen ones).

We can start doing that, and this problem would disappear. Or we could not do that, and start deprecating old migrations.